File: app/src/org/koreader/launcher/device/freescale/NTXNewEPDController.java
/* Tested on Tolino Vision2/Shine3 and a Nook Glowlight 3. */

package org.koreader.launcher.device.freescale;

import org.koreader.launcher.device.EPDController;

public class NTXNewEPDController extends NTXEPDController implements EPDController {
    @Override
    public void setEpdMode(android.view.View targetView,
                           int mode, long delay,
                           int x, int y, int width, int height, String epdMode)
    {
        requestEpdMode(targetView, mode, delay, x, y, width, height);
    }
}

File: app/src/org/koreader/launcher/device/freescale/NTXNewEPDController.kt
/* Tested on Tolino Vision2/Shine3 and a Nook Glowlight 3. */

package org.koreader.launcher.device.freescale

import org.koreader.launcher.interfaces.EPDInterface

class NTXNewEPDController : NTXEPDController(), EPDInterface {
    override fun setEpdMode(targetView: android.view.View,
                            mode: Int, delay: Long,
                            x: Int, y: Int, width: Int, height: Int, epdMode: String?)
    {
        requestEpdMode(targetView, mode, delay, x, y, width, height)
    }
}
